### PR TITLE
Made change on Addin 2013

### PR DESCRIPTION
**Pull request by Developer-Felix**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/management/procedures-related-to-deployment.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Your user account needs to be a member of the Operations Manager Administrators
 
 1. Log on to your SCOM server and open the SCOM console.
 
-2. Click **Monitoring** and then click **Windows Computers**.
+2. Click **Monitoring** and then click on **Windows Computers**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/management/procedures-related-to-deployment.md:30]:
```suggestion
2. Click **Monitoring** and then select **Windows Computers**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**